data-serving/data-service/api/openapi.yaml

```diff
@@ -117,6 +117,22 @@ paths:
           $ref: "#/components/responses/422"
         "500":
           $ref: "#/components/responses/500"
+  /cases/batchValidate:
```

Comment:
will we be able to remove the validate_only param on the other endpoint once we're using this?

Reply:
Yep, should -- the only current user will be migrating to the batch API.